### ISSUE TITLE
Periods (.) in export path cause error

### ISSUE THREAD

**Issue by altervais** [KosmosisDire/obsidian-webpage-export#149]
At first everything was fine, I could export in any way. Plugin is great!
Yesterday suddenly there was a following problem. 
I've tried changing vault, reinstalling the plugin, tweaking options, nothing.

**Operating System**
Windows 10

##### [Error] Could not export file: styles.md
```
Error: exportToFolder must be an absolute path to a folder: F:\Temp\altervais prompt palette v.2.0
    at new ExportFile (plugin:webpage-html-export:11649:13)
    at HTMLExportPlugin.exportFile (plugin:webpage-html-export:11815:26)
    at HTMLExportPlugin.exportFolder (plugin:webpage-html-export:11856:39)
    at async HTMLDivElement.eval (plugin:webpage-html-export:11760:24)
```
![Stu7wJ9](https://github.com/KosmosisDire/obsidian-webpage-export/assets/93525657/ff15a8a5-dfde-4883-98e3-9a04c32fd58f)

**Comment by altervais** [comment]
How did I tryed so many things and never come up with such an obvious solution! I'm embarrassed. Everything works, thank you)

**Comment by KosmosisDire** [comment]
Haha no problem. This is still a bug though so I'll make sure to fix this. Thanks! :)

**Comment by lduperval** [comment]
Following since I have the same problem and I suspected it was the period (I have 2 paths with periods)

**Comment by KosmosisDire** [comment]
This should be fixed in v1.7.0, but let me know if it is not!
